setup.cfg

```diff
@@ -14,7 +14,8 @@ exclude = kolibri/*/migrations/*
 [isort]
 atomic = true
 multi_line_output = 5
-line_length = 79
+line_length = 160
+indent='    '
```

Comment:
This is a fly-by fix that solves a problem I was having with precommit hooks -- the [isort] code was reducing indents to 2 spaces, and then flake8 was complaining about indents; this removes the catch-22.